plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -94,6 +94,10 @@ global proc mayaUsd_PrefUntexturedModeChanged()
 	if (`radioButtonGrp -exists usdPrefUntexturedMode`) {
 		int $sel = `radioButtonGrp -q -select usdPrefUntexturedMode`;
 		optionVar -iv mayaUsd_ShowDisplayColorTextureOff ($sel == 2);
+        
+        // Set a temporary optionVar saying that we have changed the untextured mode
+        // so in case user cancels, we can do reset again.
+        optionVar -transient -iv mayaUsd_PrefUntexturedModeChanged 1;
```

Comment:
transient optionvars are not saved when exiting Maya. ogs -reset is not a cheap operation so we should not just call it unless needed (such as when opening the pref dialog we don't need to call it). Use this optionvar to keep track of when the untextured mode pref changes and only call ogs -reset when needed.